set.mm

```diff
@@ -386751,6 +386752,130 @@ are congruent (CPCTC), focusing on a specific segment.  (Contributed
+          eqid crag eqidd simpr s3eqd mircl tgcgrcomlr eqtr2d tgcgrneq eqnetrrd
+          cfv cs3 mirinv necon3bid mirbtwn oveq1d tgbtwnouttr2 tgbtwncom oveq2d
+          mpbid ccgrg eqtrd mpbird eqcomd axtg5seg trgcgr ragcgr ragcom neeqtrd
+          israg axtgcgrrflx mircgr eleqtrd orcd ragcol eqeltrd btwncolg3 neneqd
+          wo ragflat pm2.65da neqned ragperp ) ACNLVEZBECJKLMONQRSTUAAEJKLCNQST
+          UAUCUHANCANBVFCBVFVGZNCVHABCDEFGHIJKLMNOPQRSTUAUBUCUDUEUFUGUHUIUJUKUL
+          UMUNUOUPUQURUSUTVAVBVCVDVIZUDNCBVJVKVOZVLUBAYHBNAECNJKLQSTUAUCUHYKVMY
+          JVNAECNJKLQSTUAUCUHYKVPAOHILVEZBAEJKLHIOQSTUAUEUFUIUMAEJKLIHOQSTUAUFU
+          EUIAHIUMVOUNVQVRULVSZYKAONAONVTZOGVTZAYNWAZHOGEJWBWOZJKLMQRSTYQWEZAJW
+          CVFYNUAWDZAHEVFYNUEWDZAOEVFYNUIWDZAGEVFYNUGWDZYPGOHEYQJKLMQRSTYRYSUUB
+          UUAYTYPCOHGEYQJKLMQRSTYRYSACEVFYNUCWDZUUAYTUUBYPCOHWPCNHWPJWFWOZYPCOH
+          HCNYPCWGAYNWHZYPHWGWIYPHNCEYQJKLMQRSTYRYSYTANEVFYNUHWDZUUCYPPNCHEYQJK
+          LMQRSTYRYSAPEVFYNUJWDZUUFUUCYTAPNCWPUUDVFZYNAUUHPCMVEZPCNYQWOWOZMVEZV
+          TAUUIPDMVEUUKACPDPEJKMQRSUAUCUJUKUJAFPYQWOZWOZODEPJKMPFOCQRSUAUSUIUCA
+          PEYQJKLUULMFQRSTYRUAUJUULWEZUSWJZUIUKUJUJAOFACHOFEJKMQRSUAUCUEUIUSAOF
+          MVEZOHMVEZCHMVEVAAHOHCEJKMQRSUAUEUIUEUCUOWKWLAHCAHBVFYIHCVHZAHYLBAEHI
+          JKLQSTUAUEUFUMVPULVSUDHCBVJVKZVOWMVOZACOFEJKMQRSUAUCUIUSACGOFEJKMQRSU
+          AUCUGUIUSAOGYQWOZWOZOVHGOVHACUVBOUPAOCAOBVFYIOCVHYMUDOCBVJVKVOZWNAUVB
+          OGOAGOEYQJKLUVAMQRSTYRUAUGUVAWEZUIWQWRXDZAGUVBOKVECOKVEAGOEYQJKLUVAMQ
+          RSTYRUAUGUVDUIWSACUVBOKUPWTVSZUTXAXBVBAOFOUUMEJKMQRSUAUIUSUIUUOAOPFWP
+          UUDVFUUPOUUMMVEVTAFPOEYQJKLMQRSTYRUAUSUJUIAHGOFEJXEWOZYQPOJKLMQRSTYRU
+          AUEUGUIUVGWEZUSUJUIAHGOWPUUDVFZHOMVEZHUVBMVEZVTAUVJHCMVEZUVKUOACUVBHM
+          UPXCXFAHGOEYQJKLMQRSTYRUAUEUGUIXNXGZAHGOFEUVGPOJMQRUVHUAUEUGUIUSUJUIA
+          GHPFEJKMQRSUAUGUEUJUSAHOPEHJKMFFOGQRSUAUSUIUGUEUIUJUEUSUUTAGOFEJKMQRS
+          UAUGUIUSUTXBUQAOFOHEJKMQRSUAUIUSUIUEVAWKAOPMVEZOGMVEURXHZAEJKMFHQRSUA
+          USUEXOAUUPUUQVAXHZXIWKAOGOPEJKMQRSUAUIUGUIUJUVOWKUVPXJXKXLAOPFEYQJKLM
+          QRSTYRUAUIUJUSXNXDWKAODMVEZOCMVEZVCXHZAPFPUUMEJKMQRSUAUJUSUJUUOAPUUMM
+          VEPFMVEAPFEYQJKLUULMQRSTYRUAUJUUNUSXPXHWKAUVNWGXIWKADUUJPMVDXCXFAPNCE
+          YQJKLMQRSTYRUAUJUHUCXNXGWDYPPONAPOVHYNAOPAOGOPEJKMQRSUAUIUGUIUJUVOAGO
+          UVEVOZWMVOWDUUEXMYPPNHLVEZVFNHVTYPPOHLVEUWAYPEJKLOHPQSTYSUUAYTUUGYPHO
+          YPHCHOEJKMQRSYSYTUUCYTUUAAUVLUVJVTYNAUVJUVLUOXHZWDAUURYNUUSWDWMVOAPHO
+          LVEVFYNAEJKLHOPQSTUAUEUIUJAHCHOEJKMQRSUAUEUCUEUIUWBUUSWMUQVQWDVRYPONH
+          LUUEWTXQXRXSXLXTACOVHYNUVCWDACOGLVEVFYOYCYNAEJKLOGCQTSUAUIUGUCACGOEJK
+          MQRSUAUCUGUIUVFXBYAWDXSXLAUVIYNUVMWDYDYPOGAOGVHYNUVTWDYBYEYFAONCEYQJK
+          LMQRSTYRUAUIUHUCAONCWPUUDVFUVROUUJMVEZVTAUVRUVQUWCUVSADUUJOMVDXCXFAON
+          CEYQJKLMQRSTYRUAUIUHUCXNXGXLYG $.
+      $}
+
+      $( Lemma for ~ foot : existence part.  (Contributed by Thierry Arnoux,
```

Comment:
I would not describe this as a lemma (and its label has, and justifiably so, no suffix "lem"), but with a full text description:
"From a point ` C ` outside of a line ` A ` , there exists a point ` x ` on ` A ` such that ` ( C L x ) ` is perpendicular to ` A ` .
This point is unique, see ~ foot . "